reflex/.templates/web/utils/state.js

```diff
@@ -534,7 +507,17 @@ export const useEventLoop = (
       })()
     }
   })
-  return [state, addEvents, connectError]
+
+  // Route after the initial page hydration.
+  useEffect(() => {
+    const change_complete = () => addEvents(initialEvents())
```

Comment:
```suggestion
    const change_complete = () => addEvents(initial_events())
```

probably should use the function local var here